modcc/debug-CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+# The modcc compiler is split into a staric library and driver;
```

Comment:
Should this file be deleted before merging? It looks like a backup copy.

Reply:
It totally should. Mismatch between my source tree and the commit tree; I'll remove it.